R/subscript-loc.R

```diff
@@ -58,23 +58,42 @@ vec_as_location <- function(i,
   if (!missing(...)) ellipsis::check_dots_empty()
 
   i <- vec_as_subscript(i, arg = arg)
-  .Call(vctrs_as_location, i, n, names, "invert")
+  .Call(
+    vctrs_as_location,
+    i = i,
+    n = n,
+    names = names,
+    loc_negative = "invert",
+    loc_oob = "error"
+  )
 }
 #' @rdname vec_as_location
 #' @param negative Whether to `"invert"` negative values to positive
 #'   locations, throw an informative `"error"`, or `"ignore"` them.
+#' @param oob If `"error"`, throws an informative `"error"` if some
+#'   elements are out-of-bounds. If `"extend"`, out-of-bounds
+#'   locations are allowed if they are consecutive after the end. This
+#'   can be used to implement extensible vectors like `letters[1:30]`.
```

Comment:
Is "extensible" right? "extendable" maybe? I found this a little confusing

Reply:
"stretchable" vectors?